File: Eugenics/Eugenics/Models/Skill.cs
namespace Eugenics.Models
{
    public class Skill
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string ActivationStat { get; set; }
        public float? ActivationMultiplier { get; set; }
        public bool DLC { get; set; }
    }
}

File: Eugenics/Eugenics/Scripts/models/skill.js
(function () {
    "use strict";

    define([
        "knockout"
    ], function (ko) {
        var SkillViewModel = function (skill) {
            this.id = ko.observable(skill.id);
            this.name = ko.observable(skill.name);
            this.description = ko.observable(skill.description);
            this.activationStat = ko.observable(skill.activationStat);
            this.activationMultiplier = ko.observable(skill.activationMultiplier);
            this.dlc = ko.observable(skill.dlc);
        };

        return SkillViewModel;
    });
})();
